rules/windows/execution_suspicious_dotnet_compiler_parent_process.toml

```
@@ -0,0 +1,41 @@
+[metadata]
+creation_date = "2020/08/21"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/08/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Suspicious dotNet Code execution. Verify process details such as commandline, parent process and network
```

```suggestion
Identifies suspicious .NET code execution.
```

I'd recommend moving this to the `note` field, as it pertains to triaging/investigating the alert.

> Verify process details such as commandline, parent process and network